tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -30,6 +31,12 @@ public class AspNetBase : TestHelper
     {
         protected const string DefaultAttackUrl = "/Health/?arg=[$slice]";
         private const string Prefix = "Security.";
+        private static readonly Regex AppSecWafDuration = new(@"_dd.appsec.waf.duration: \d+\.0", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex AppSecWafDurationWithBindings = new(@"_dd.appsec.waf.duration_ext: \d+\.0", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex AppSecWafVersion = new(@"\s*_dd.appsec.waf.version: \d.\d.\d,?", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex AppSecRuleVersion = new(@"\s*_dd.appsec.event_rules.version: \d.\d.\d,?", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex AppSecEventRulesLoaded = new(@"\s*_dd.appsec.event_rules.loaded: \d+\.0,?", RegexOptions.IgnoreCase | RegexOptions.Compiled);
+        private static readonly Regex AppSecErrorCount = new(@"\s*_dd.appsec.event_rules.error_count: 0.0,?", RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
Is it possible to test for the `"_dd.appsec.event_rules.errors"` tag? I think that's the only new tag/metric that's not represented here

Reply:
yes it's part of the test `[AspNetCore5WafInitialization](https://github.com/DataDog/dd-trace-dotnet/blob/anna/appsec/update-waf-and-ruleset-1.3.0/tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetCore5WafInitialization.cs).cs` and its [snapshots](https://github.com/DataDog/dd-trace-dotnet/blob/297eca7993445321247d43d875f62b42f3c1c724/tracer/test/snapshots/Security.AspNetCore5WafInitialization.__enableSecurity%3DTrue_expectedStatusCode%3D200_ruleset%3Dwrong-tags-name-rule-set.json.verified.txt#L29)

Just did it for NETCOREAPP3_0_OR_GREATER because there's no different code path in other versions, it's mostly to test waf init errors and the report of it, which doesnt differ in other versions.
These regexes in `AspNetBase.cs` are more to scrub them from other tests as these aren't always deterministic

Reply:
Ah thanks, I missed its usage in the snapshots 👍🏼